docs/dev/system_keyspace.md

```diff
@@ -33,6 +33,7 @@ CREATE TABLE system.large_partitions (
     partition_size bigint,
     partition_key text,
     range_tombstones bigint,
+    dead_rows bigint,
```

Comment:
hmm, these two columns were added last, after `compaction_time` (they have to, for backward compatibility). So the doc should reflect that.

Reply:
edit: schema sorts the columns by name, not by order of insertion to schema_builder.
Backward compatibility is achieved by matching column names, not column id's (for better or worse)